platforms/windows/SQLitePlugin/SQLitePlugin.cpp

```diff
@@ -307,13 +307,18 @@ namespace SQLitePlugin
                     {
                         try
                         {
-                            CopyDbAsync(assetFile, to_hstring(*dbFileName)).GetResults();
+                            CopyDbAsync(assetFile, to_hstring(*dbFileName)).get();
                         }
                         catch (hresult_error const& ex)
                         {
-                            // CopyDbAsync throws when the file already exists.
-                            onFailure("Unable to copy asset file: " + winrt::to_string(ex.message()));
-                            return;
+                            if (ex.code() == 0x800700B7) 
```

Comment:
```suggestion
                            if (ex.code() == HRESULT_FROM_WIN32(ERROR_ALREADY_EXISTS)) 
```
I _think_ this should be the HRESULT you're looking for, without having to hardcode the error. And I _think_ that comes from winerror.h.

If that doesn't work just include a comment to make it clear which HRESULT you're using here.

Reply:
Excellent, thank you once again. I'll do a step-through and confirm and commit tonight